SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -16410,6 +16410,9 @@ void CNSSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_container
     /*--- Check if the node belongs to the domain (i.e, not a halo node) ---*/
     
     if (geometry->node[iPoint]->GetDomain()) {
+
+      /* If it is a customizable patch, retrieve the specified wall heat flux. */
+      if (config->GetMarker_All_PyCustom(val_marker)) Wall_HeatFlux = geometry->GetCustomBoundaryHeatFlux(val_marker, iVertex);
```

Comment:
It would be great to get these features into the incompressible solver too, but we can do that in a future PR (some changes are coming there too).

Reply:
Sure ! In the future, the features should be extended to other solvers like the incompressible and also be extended to other kinds of BCs (typically inlet velocity profile).

Reply:
Any updates on this? (Specifically, custom BCs for Euler/Navier-Stokes solvers). A python-based implementation would be much simpler than the C++ based implementation I was creating.

If there aren't any updates, I could work on the implementation myself.  It looks fairly straightforward.

Reply:
There is no update since the last merge. I won't be able to work on it intensively before a long time (projects and a thesis to finish...) so if you are interested you could definitely work on it. As you said, it should be straightforward, at least up to a certain point.